paddle/fluid/inference/tests/api/analyzer_transformer_tester.cc

```diff
@@ -183,17 +183,27 @@ void SetInput(std::vector<std::vector<PaddleTensor>> *inputs) {
 }
 
 // Easy for profiling independently.
-TEST(Analyzer_Transformer, profile) {
+void profile(bool use_mkldnn = false) {
   AnalysisConfig cfg;
   SetConfig(&cfg);
   std::vector<PaddleTensor> outputs;
+  if (use_mkldnn) {
+    cfg.EnableMKLDNN();
+    std::unordered_set<std::string> op_list = {"transpose2"};
+    cfg.SetMKLDNNOp(op_list);
```

Comment:
Why should we specify op_list for transformer ut? Could we remove line192-193?

Reply:
@luotao1 Good point. Manual selection of mkl-dnn ops for transformer was removed.